### PR TITLE
Feature/typing animation

### PR DESCRIPTION
**Pull request by imtoori**
# Submit a pull request

## CLA

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required).
- [x] The code changes follow best practices
- [x] Code changes are tested (add some information if not applicable)

### CODE REVIEW

File: example/lib/main.dart [hunk 1]
```diff
@@ -223,7 +223,9 @@ class ChannelPage extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      appBar: ChannelHeader(),
+      appBar: ChannelHeader(
+        showTypingIndicator: false,
+      ),
```
Comment (xsahil03x) [inline example/lib/main.dart:228]: Is this correct as per Figma designs?

Comment (imtoori) [reply example/lib/main.dart:228]: yep, I checked with Francesco
We support typing indicators in channelheader, but in the sample app we show them at the bottom of the messagelistview, you can check on figma

Comment (xsahil03x) [reply example/lib/main.dart:228]: Cool, LGTM then.

**Review by xsahil03x — COMMENTED** [review]
LGTM

**Pull request merged**